modules/accounts/calendar.nix

```diff
@@ -129,6 +131,7 @@ in {
   options.accounts.calendar = {
     basePath = mkOption {
       type = types.str;
+      default = ".calendar";
```

Comment:
Not sure I like this, I would rather have the user know where the files are. Maybe move it to "example"

Reply:
Agreed, the error message without an default value should be good enough as a hint to lookup configuration options: `error: The option 'accounts.calendar.basePath' is used but not defined.`